### DISCUSSION TITLE
create database SecondaryDB as replica ?

### DISCUSSION THREAD

**Discussion by extdponomarchuk** [Snowflake-Labs/terraform-provider-snowflake#947] [Ideas]
Hi there,
Is it possible to implement the following REPLICATION statements in this terraform provider?

```
-- Promote an existing database in your local account as primary
alter database PrimaryDB enable replication to accounts AZURE_EASTUS2.AcmeProviderAccount2;

-- Replicate your existing database to another region
create database SecondaryDB
as replica of AWS_US_WEST_2.AcmeProviderAccount1.PrimaryDB;
```

according to the manual https://docs.snowflake.com/en/user-guide/secure-data-sharing-across-regions-plaforms.html

**Comment by rsubudhi** [comment]
On you second point you can manage through terraform 
resource "snowflake_database" "from_replica" {
  name                        = "testing_3"
  comment                     = "test comment"
  data_retention_time_in_days = 3
  from_replica                = "org1\".\"account1\".\"primary_db_name"
}

https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/resources/database

first point still need to execute manually
